mne/io/nirx/nirx.py

```diff
@@ -77,7 +77,7 @@ class RawNIRX(BaseRaw):
 
     @verbose
     def __init__(self, fname, saturated, preload=False, verbose=None):
-        from ...externals.pymatreader import read_mat
+        from scipy.io import loadmat
```

Comment:
Why `loadmat` here and not `pymatreader`? And if you want `loadmat` then this could be imported at the top.

Reply:
> Why loadmat here and not pymatreader? 

Don't want to require `pymatreader` if we don't have to, and it was easy enough to change it here

> And if you want loadmat then this could be imported at the top.

No, most `scipy` imports should be nested for speed. We check that implicitly here:

https://github.com/mne-tools/mne-python/blob/7ecd46fe444a9c7e9d2b0c7f63ef05af09d41082/mne/tests/test_import_nesting.py#L16-L22

Reply:
> You mean to decrease the time it takes to import mne as opposed to running that specific function?

Yep

> This particular example would not affect that, because we do not export the io module into the main namespace, right?

We import almost everything, including `io`

https://github.com/mne-tools/mne-python/blob/7ecd46fe444a9c7e9d2b0c7f63ef05af09d41082/mne/__init__.py#L114

Reply:
Right. OK then!